src/generic/grideditors.cpp

```diff
@@ -441,38 +441,17 @@ void wxGridCellTextEditor::SetSize(const wxRect& rectOrig)
 
     // Make the edit control large enough to allow for internal margins
     //
-    // TODO: remove this if the text ctrl sizing is improved esp. for unix
+    // TODO: remove this if the text ctrl sizing is improved
```

Comment:
I also hoped we could get rid of this TODO... Why is it still necessary and why do we still need platform-specific code here?